mobile_back_tflite/cpp/backend_tflite/ios/BUILD

```diff
@@ -0,0 +1,22 @@
+load("@build_bazel_rules_apple//apple:apple.bzl", "apple_xcframework")
+
+apple_xcframework(
+    name = "libtflitebackend",
```

Comment:
I created a separate file for apple tflite target to ensure that there are no name conflicts.